taxcalc/functions.py

```diff
@@ -1127,14 +1141,15 @@ def ExpEarnedInc(_exact, c00100, CDCC_ps, CDCC_crt,
 
 
 @iterate_jit(nopython=True)
-def RateRed(c05800, _othtax, _exact, x59560, _earned):
+def RateRed(c05800, _othtax, _exact, e59560, _earned, c59660):
 
     # rate reduction credit for 2001 only, is this needed?
     c05800 = c05800
     c07970 = 0.
+    c59560 = 0.
 
     if _exact == 1:
-        c59560 = x59560
+        c59560 = e59560
```

Comment:
_exact == 1 just means that we do the calculations with e values instead of c values. We should keep using c values here. That said, RateRed credit is outdated, so *i think* we could remove this entire function without changing anything.